File: seller/src/main/java/com/example/seller/SplashScreen.java
package com.example.seller;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import androidx.appcompat.app.AppCompatActivity;
import com.example.seller.Login.ui.main.LoginActivity;

public class SplashScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        //getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
        SessionManager sessionManager = new SessionManager(this);
        new Handler().postDelayed(() -> {
            if (sessionManager.checkLogin()) {
                Intent intent = new Intent(SplashScreen.this, HomeActivity.class);
                startActivity(intent);
                finish();
            } else {
                Intent intent = new Intent(SplashScreen.this, LoginActivity.class);
                startActivity(intent);
                finish();
            }
        }, 1000);
    }
}

File: app/src/main/java/com/example/loginApp/SplashScreen.kt
package com.example.loginApp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler

class SplashScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        val sessionManager = SessionManager(this)
        Handler().postDelayed({
            if (sessionManager.checkLogin()) {
                val intent = Intent(applicationContext, MainActivity23::class.java)
                startActivity(intent)
                finish()
            } else {
                val intent = Intent(applicationContext, MainPage::class.java)
                startActivity(intent)
                finish()
            }
        }, 1000)
    }
}